### DISCUSSION TITLE
v4.0.0-beta.1

### DISCUSSION THREAD

**Discussion by mattieserver** [mattieserver/netbox-topology-views#505] [Announcements]
## What's Changed
* 477 prepare for netbox 4.0 by @mattieserver in https://github.com/netbox-community/netbox-topology-views/pull/495

**Full Changelog**: https://github.com/netbox-community/netbox-topology-views/compare/v3.9.1...v4.0.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/netbox-community/netbox-topology-views/releases/tag/v4.0.0-beta.1'>v4.0.0-beta.1</a>.</em>

**Comment by milevskiy2** [comment]
Could you please tell me when the release will be?

**Reply by dooziedan** [reply]
I think there is no planned release date:

https://github.com/netbox-community/netbox-topology-views/issues/477#issuecomment-2129614840

> Hi all,
>
> I completely understand why you are asking impatiently, but everything there is to say about this is already written in the first post above. There are all the points that still need to be done and where to find the 4.0 branch.
>
> There is no ETA because we are not rushing. This has been answered already. I myself would also like to have the upgrade ready and upgrade my NetBox version to 4.0. However, there are a few issues why it is taking so long this time:
>
> We are a very small team of only two people.
> I am currently working a lot and have little free time (I think the same applies to Mattijs).
> The weather is great and there's little reason to sit in a stuffy room in front of a screen instead of enjoying the nature outside.
> If you need it to be done faster, contribute to the project. Any help is very welcome.

**Reply by mattieserver** [reply]
You can use the beta for now.

Currently there are still 2 things that we know that do not work. I have created the needed fixes so we might release later in the week.
